### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by zealott6** [Chia-Network/chia-blockchain#3499] [Support]
The truth is that I am already tired of this synchronization problem at the moment it only desynchronizes and does not synchronize again and this removes the chances of winning, if this continues I think I will abandon it since I am thinking that it is designed so that they win only to some chosen people as i dont understand why it got out of sync for no reason
![programa de basura](https://user-images.githubusercontent.com/83084021/116834619-8f1ce380-ab84-11eb-94a3-22d35b9d4b30.png)

**Comment by vm8net** [comment]
这确实很垃圾，而且是大规模出现同步不了的现象，而这种现象发生在全世界，不是区域性的。

**Comment by hefenglim** [comment]
same issue, i have meet lot of time. need to restart CHIA and machine. it not stable.

**Comment by zealott6** [comment]
It is silly, because it makes you waste your time, I have realized that I have been many kisses up to 10 hours out of sync and it is very annoying because it takes away your chances of winning

**Comment by sargonas** [comment]
try manually adding the following nodes to see if that kickstarts things for you:

chiadb.net:8444, node-or.chia,net:8444, and node-eu.chia.net:8444

Also make sure your firewall is open on port 8444 to port forward to the computer running the app, or baring that try enabling UPnP on your router.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
